tests/ghbug335.phpt

```diff
@@ -0,0 +1,30 @@
+--TEST--
+GH Bug #335: APCu stampede protection is broken
+--SKIPIF--
+<?php require_once(dirname(__FILE__) . '/skipif.inc'); ?>
+--INI--
+apc.enabled=1
+apc.enable_cli=1
+apc.serializer=default
```

Comment:
This line shouldn't be necessary, as this test does not rely on any particular serializer. Leaving out the option here will make sure that CI tests both default and php serializers.